code/game/turfs/simulated/wall/mineral_walls.dm

```diff
@@ -303,3 +303,19 @@
 /turf/closed/wall/mineral/plastitanium/copyTurf(turf/T)
 	. = ..()
 	T.transform = transform
+
+/////////////////////Pirate Plastitanium walls/////////////////////
+
+/turf/closed/wall/mineral/plastitanium/pirate
```

Comment:
I told you to make this a subtype of `/turf/closed/wall/r_wall/syndicate`.
Also remove the `explosion_block` value of 50, the parent type's 20 will suffice.